tide_core.module

```diff
@@ -101,3 +101,23 @@ function tide_core_entity_operation(EntityInterface $entity) {
 
   return $operations;
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function tide_core_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (!isset($form['field_node_primary_site_target_id']) && !isset($form['moderation_state'])) {
+    return;
+  }
+  if (isset($form['moderation_state']['#options']['Editorial']['editorial-published'])) {
```

Comment:
@vincent-gao Could you please add a comment to describe this check is necessary.